### DISCUSSION TITLE
Custom local Images for Comparison Slider

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [upscayl/upscayl#1024] [Q&A] [answered]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have searched for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Description

Options where i can select two images from local storage and upscayl will show comparison slider for them.

I was trying the custom model but model was not being read by upscayl, so i ran the model from command-line and got the output but had no way to compare them locally.

An option where upscayl will lets user choose custom image would be awesome as it will add another use case for it.

**Comment by aaronliu0130** [comment, marked as answer]
https://github.com/kylewetton/image-compare-viewer
